Felienne/hedy
# [FEATURE] Enables dynamic keyword changing from level defaults

**Description**
In this PR we start with the implementation of supporting level defaults and adventures with localized keywords. Currently we only support the manual switching of keywords by the user. However, with the implementation of the feature to enable users to set their preferred keyword language it would be nice if all code snippets are automatically shown in this language. This PR is a wip to discuss if we are on the right track.

Currently the level defaults for `nl` and `en` are adjusted to support the dynamic changing of the keywords. We replace each keyword with a placeholder, for example: `{print} hello world!`. On runtime we replace the placeholder with the current keyword language. If there is a keyword language set in the profile we hide the switcher as well.

The `test_z_defaults` have been a bit bucket-fixed to make sure they pass. We now parse the code as well in the test files to make sure the code snippets contain valid code and pass the tests. We might have to re-write the tests somewhere in the future to make sure it all keeps working when we expand on this work.

**How to test**
Make sure you have a preferred keyword language set in your profile. To notice the change set it too `nl` and navigate to level 5. Notice that the level default code is correctly parsed to contain dutch keywords.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
